src/utilities/methods.js

```diff
@@ -119,9 +119,19 @@ export const getEdgeAgentStatusCode = (code, t) => {
 
 /** Converts a packageType enum to a translated string equivalent */
 export const getPackageTypeTranslation = (packageType, t) => {
```

Comment:
Consider refactoring to use getEnumTranslation.

Reply:
The two methods have different behavior for default case, so it's not possible to do this.

Reply:
Your two functions could remain... and utilize the shared function internally. I specifically added the defaultVal param to allow for this case.

```js
/** Converts a packageType enum to a translated string equivalent */
export const getPackageTypeTranslation = (packageType, t) => getEnumTranslation(t, 'packageType', packageType, t('packageType.unknown'));

/** Converts a packageType enum to a translated string equivalent */
export const getConfigTypeTranslation = (configType, t) => getEnumTranslation(t, 'configTypes', configType, configType);
```

Reply:
done